### DISCUSSION TITLE
[@types/node] test runner mocking types

### DISCUSSION THREAD

**Discussion by nyonson** [DefinitelyTyped/DefinitelyTyped#63575] [Issues with a @types package]
The nodejs builtin test runner gained support for [mocking](https://nodejs.org/api/test.html#mocking) in v19.1, but these types are not defined in @types/node.

[a comment by an automated account is not shown]

**Reply by peterblazejewicz** [reply]
Types are still at v18, so that's a future change

**Reply by cysp** [reply]
For interest, it seems that mocking was released on the v18 branch as well, in v18.13.0.
https://nodejs.org/uk/blog/release/v18.13.0/#support-function-mocking-on-node-js-test-runner
https://nodejs.org/dist/latest-v18.x/docs/api/test.html#mocking
https://nodejs.org/dist/latest-v18.x/docs/api/test.html#class-mocktracker
